joint_state_publisher/joint_state_publisher/joint_state_publisher

```diff
@@ -386,7 +384,6 @@ class JointStatePublisherGui(QWidget):
         self.maxrowsupdown.setMinimum(1)
         self.maxrowsupdown.setMaximum(len(sliders))
         self.maxrowsupdown.setValue(self.num_rows)
-        self.maxrowsupdown.lineEdit().setReadOnly(True)  # don't edit it by hand to avoid weird resizing of window
```

Comment:
@bmagyar's comment says this is read only to avoid weird window resizing, but with it removed I didn't see the window resize when entering different numbers. The joint sliders would resize as the number of columns changed, but that happens when clicking the arrow buttons too.